Appraisals

```diff
@@ -0,0 +1,7 @@
+appraise "rails-4" do
+  gem "rails", "4.2.7"
```

Comment:
This should be 4.2.7.1 as the .1 is an important security update which most people will have installed.